Login/src/main/resources/application.properties

```diff
@@ -1 +1,14 @@
 server.port=8081
+spring.datasource.url=jdbc:mysql://bms.cwdujd64fd6x.us-west-1.rds.amazonaws.com:3306/bms
```

Comment:
Create a application_mock.properties with dummy values. Do not store and commit any links